includes/Traits/External_Utils.php

```diff
@@ -0,0 +1,289 @@
+<?php
+/**
+ * Trait WordPress\Plugin_Check\Traits\External_Utils
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Traits;
+
+/**
+ * Trait for find files php,js,css.
+ *
+ * @since 1.0.0
+ */
+trait External_Utils {
+	/**
+	 * Filter the given array of files for php,js,css files.
+	 *
+	 * @since 1.4.0
+	 *
+	 * @param array  $files                Array of file files to be filtered.
+	 * @param string $plugin_relative_path Plugin relative path.
+	 * @return array An array containing php,js.css files, or an empty array if none are found.
+	 */
+	protected function filter_files_for_external( array $files, $plugin_relative_path ) {
```

Comment:
We already have `filter_files_by_extensions()` method to filter files by extensions.